### DISCUSSION TITLE
Octopus tracker?

### DISCUSSION THREAD

**Discussion by Bigsy** [AmyJeanes/TeslaMateAgile#85] [General]
I think I already know the answer is probably but this doesn't support pulling Octopus tracker tariff costs? https://octotrak.com/

**Comment by AmyJeanes** [comment]
It does support Octopus Agile but not currently the Octopus Tracker tariff unfortunately.

Looks like the API works in daily increments as far as I can tell but I'm not sure if it is currently possible to retrieve the data, the API documentation is completely missing for the tracker endpoints for some reason.

I tried to send an email to the maintainer of Octotrak to see if they had any documentation or would let me use their API, but the email bounced with a non-delivery report so I've tried to send it to the WHOIS domain owner email, but we'll see what happens there.

If you have / can find any useful API documentation on this, let me know and I'd be happy to implement this if we can get some.
